src/bugsnag.js

```diff
@@ -1253,6 +1258,9 @@
     self.leaveBreadcrumb({ type: "navigation", name: "Bugsnag Loaded" });
   }
 
+  // Log that we've successfully loaded
+  log("JavaScript error reporting initialized (https://bugsnag.com)", false);
```

Comment:
Maybe we could also include some other diagnostic information, such as release stage and whether or not autonotify is enabled